test/test-execute/exec-runtimedirectory-stdoutput.service

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+[Unit]
+Description=Test for RuntimeDirectory used by StandardOutput
+
+[Service]
+ExecStart=sh -c 'printf "hello\nhello\n"'
+Type=oneshot
+RuntimeDirectory=test-exec_runtimedirectory_standardoutput
```

Comment:
Please set `RuntimeDirectoryPreserve=yes`, and check the contents of the output file.